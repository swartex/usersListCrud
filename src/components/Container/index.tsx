import { FC, PropsWithChildren } from "react";

const Container: FC<PropsWithChildren> = ({ children }) => {
  return <div className="max-w-7xl mx-auto">{children}</div>;
};

export default Container;
