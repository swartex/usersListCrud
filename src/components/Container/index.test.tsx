import { render } from "@testing-library/react";
import Container from "./index";

describe("Container", () => {
  it("renders", () => {
    const { asFragment } = render(<Container>Hello World!</Container>);

    expect(asFragment()).toMatchSnapshot();
  });

  it("should render children", () => {
    const { getByText } = render(<Container>Hello World!</Container>);

    expect(getByText("Hello World!")).toBeInTheDocument();
  });
});
