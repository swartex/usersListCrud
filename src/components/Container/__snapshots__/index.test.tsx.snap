// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Container > renders 1`] = `
<DocumentFragment>
  <div
    class="max-w-7xl mx-auto"
  >
    Hello World!
  </div>
</DocumentFragment>
`;
